WooCommerce/Classes/POS/Models/PointOfSaleSettingsService.swift

```diff
@@ -0,0 +1,95 @@
+import Foundation
+import struct Yosemite.SiteSetting
+import enum Yosemite.SettingAction
+import enum Yosemite.Plugin
+import class Yosemite.PluginsService
+import Observation
+
+@Observable final class PointOfSaleSettingsService {
+    private(set) var receiptStoreName: String?
+    private(set) var receiptStoreAddress: String?
+    private(set) var receiptStorePhone: String?
+    private(set) var receiptStoreEmail: String?
+    private(set) var receiptRefundReturnsPolicy: String?
+    private(set) var isLoading: Bool = false
+    private(set) var shouldShowReceiptInformation: Bool = false
+
+    private var siteID: Int64 {
+        ServiceLocator.stores.sessionManager.defaultSite?.siteID ?? 0
+    }
+
+    var storeName: String {
+        guard let site = ServiceLocator.stores.sessionManager.defaultSite else {
+            return Localization.storeNotSet
+        }
+        return site.name
+    }
```

Comment:
❓ Is the store name configurable in POS settings? If so, how is the store name used in POS?

Reply:
nit: would be nice avoiding ServiceLocator in POS, as we're hoping to modularize it one day (Povilas' HACK Week) and it'd be easier if dependencies are DI'ed.

Reply:
These are currently read-only, and ideally will remain as such for now, so can be read but cannot be changed from POS, only from the app. 

It can be found now in `PointOfSaleSettingsController` as: 

```
defaultSiteName: String? = ServiceLocator.stores.sessionManager.defaultSite?.name,
```

Let me know if that's what you had in mind.